lib.php

```diff
@@ -812,8 +812,8 @@ function plagiarism_vericite_get_settings() {
     }
 }
 
-function plagiarism_vericite_get_setting_boolean($plagiarismsettings, $key) {
-    return isset($plagiarismsettings[$key]) ? $plagiarismsettings[$key] : false;
+function get_setting_boolean($plagiarismsettings, $key, $default = false) {
```

Comment:
you were correct with the "plagiarism_vericite_" prefix before, this is a requirement to prefix all functions like this in Moodle plugins, please switch back